src/components/side-panel/interaction-panel/tooltip-config.js

```diff
@@ -107,12 +132,52 @@ function TooltipConfigFactory(DatasetTag) {
               />
             </SidePanelSection>
           ))}
+          <CompareSwitchWrapper>
+            <FormattedMessage id="compare.modeLabel" />
+            <Switch
+              checked={config.compareMode}
+              id="compare-mode-toggle"
+              onChange={() => {
+                const newConfig = {
+                  ...config,
+                  compareMode: !config.compareMode
+                };
+                onChange(newConfig);
+              }}
+              secondary
+            />
+          </CompareSwitchWrapper>
+          <SidePanelSection>
+            <PanelLabel>
+              <FormattedMessage id="compare.typeLabel" />
+            </PanelLabel>
+            <ItemSelector
+              disabled={!config.compareMode}
+              displayOption={d =>
+                intl.formatMessage({
+                  id: `compare.types.${d}`
+                })
+              }
+              selectedItems={config.compareType || COMPARE_TYPES.DECIMAL}
+              options={Object.values(COMPARE_TYPES)}
+              multiSelect={false}
+              searchable={false}
+              inputTheme={'secondary'}
```

Comment:
you can use `getOptionValue={d => d}` to have the `onChange` function return the untranslated value. hence no need to do things as below